vital/types/metadata.cxx

```diff
@@ -223,6 +224,14 @@ ::add( std::unique_ptr< metadata_item >&& item )
 #else
   this->m_metadata_map[ tag ] = item_ptr{ item.release() };
 #endif
+  if( !add_internal( *item ) )
+  {
+    std::stringstream ss;
+    ss  << demangle( typeid( *this ).name() ) << ": tag "
+        << tag_traits_by_tag( tag ).enum_name()
+        << " not supported by internal data structure";
+    LOG_DEBUG( get_logger( "metadata" ), ss.str() );
```

Comment:
This is probably going to happen very frequently for some fields in the KLV backend case, and it will generate a lot of debug messages.  I'd suggest using `LOG_TRACE` instead.